test/unit_test/passes/onnx/test_quantization.py

```diff
@@ -52,6 +52,20 @@ def test_quantization(calibrate_method, tmp_path):
     assert out is not None
 
 
+def test_dynamic_quantization(tmp_path):
+    input_model = get_onnx_model()
+    config = {"quant_mode": "dynamic"}
+    p = create_pass_from_dict(OnnxQuantization, config, disable_search=True)
+
+    if version.parse(OrtVersion) >= version.parse("1.17.0"):
```

Comment:
do we need change it to 
> 1.16.0 and < 1.18.0 

In this case, the ort-nightly would fail

Reply:
update to check for 1.17.x